### PR TITLE
some additions

### PR DESCRIPTION
**Pull request by MsAlPatina**

### CODE REVIEW

File: cdp1802app/src/MemoryView.js [hunk 1]
```diff
@@ -139,14 +149,117 @@ export default function MemoryView(props) {
                 <StyledTableCell key={`rowstr${y}`} >{end_string}</StyledTableCell>
             </StyledTableRow>
         );
-    }
-    
+    }  
+	
+	 
+	 var popupseveraty = "success";
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:164]: Don't use vars, use let and const. For details look for ES6 documentation

File: cdp1802app/src/MemoryView.js [hunk 2]
```diff
@@ -139,14 +149,117 @@ export default function MemoryView(props) {
...
+	 
+	 
+	 const [hidden, setHidden] = useState(true);
+	
+	
+  const [checked, setChecked] = useState(false);
+  const switchHandler = (event) => {
+    setChecked(event.target.checked);
+  };
+  var checkedIN2 = checked;
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:278]: Don't use vars

File: cdp1802app/src/MemoryView.js [hunk 3]
```diff
@@ -139,14 +149,117 @@ export default function MemoryView(props) {
...
+	  popupseveraty = "warning";
+		popupinfo = "RAM cant be 0 or bellow 0";
+	  ErrorSeverity = "error";
+	  ErrorInfo = "RAM cant be 0 or bellow 0";
+	  
+	  
+  }
+  
+  if(parseInt(ROM,10) > parseInt(Ram,10)){
+
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:225]: Fix the white spaces, please be consistent

File: cdp1802app/src/RAMSYSTEM.js [hunk 4]
```diff
@@ -18,16 +18,42 @@ import IconButton from '@mui/material/IconButton';
...
 
-export default function CircularIndeterminate() {
-  return (
-    <Box sx={{ display: 'flex' }}>
-      <CircularProgress />
-    </Box>
-  );
+
+
+
```
Comment (dlt-v) [inline cdp1802app/src/RAMSYSTEM.js]: Use consistent white spaces, have you thought of using a formatter?

File: cdp1802app/src/MemoryView.js [hunk 5]
```diff
@@ -22,8 +22,18 @@ import CloseIcon from '@mui/icons-material/Close';
 import TextField from '@mui/material/TextField';
 import {useRef} from 'react';
 import {useState} from 'react';
+import Switch from '@mui/material/Switch';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import RAMSYSTEM from "./RAMSYSTEM.js";
+
+  export const checkedIN = checkedIN2;
+  var checkedIN2 = "disabled";
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:34]: Don't use vars, be consistent

File: cdp1802app/src/MemoryView.js [hunk 6]
```diff
@@ -139,14 +149,117 @@ export default function MemoryView(props) {
...
 
+
+
+    console.log('value is:', event.target.value);
+  };
+  
+  const [INSW, setINSW] = useState(''); //first is the data holding var, second one is used by (event.target.value)
+	const INSWhandleChange = event => {
+    setRam(event.target.value);
+
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:193]: Why some many white spaces? Fix it

File: cdp1802app/src/MemoryView.js [hunk 7]
```diff
@@ -139,14 +149,117 @@ export default function MemoryView(props) {
...
+	  ErrorInfo = "RAM cant be 0 or bellow 0";
+	  
+	  
+  }
+  
+  if(parseInt(ROM,10) > parseInt(Ram,10)){
+  
+  
+ 
+		 popupseveraty = "error";
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:228]: consistent tabs please

**Review by dlt-v — CHANGES_REQUESTED** [review]
Please check code comments.

File: cdp1802app/src/MemoryView.js [hunk 8]
```diff
@@ -26,13 +26,22 @@ import Switch from '@mui/material/Switch';
...
 
   export const checkedIN = checkedIN2;
+  
+  
   var checkedIN2 = "disabled";
   //const RAMSYSTEM = require("./RAMSYSTEM");
   let ErrorSeverity = "success";
   let ErrorInfo = "success";
-
+
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:38]: white space

File: cdp1802app/src/MemoryView.js [hunk 9]
```diff
@@ -337,6 +362,10 @@ export default function MemoryView(props) {
 }
 export {ErrorInfo};
 export {ErrorSeverity};
+
```
Comment (dlt-v) [inline cdp1802app/src/MemoryView.js:365]: spaces

File: cdp1802app/src/RAMSYSTEM.js [hunk 10]
```diff
@@ -24,36 +24,33 @@ import {useState} from 'react';
...
+        { field: 'id', headerName: 'adress', width: 70 },
+        { field: 'firstname', headerName: '0', width: 130 },
+        { field: 'lastName', headerName: '1', width: 130 },
+        { field: 'userName', headerName: '2', width: 130},
+        { field: 'userEmail', headerName: '3', width: 180 },
+        { field: 'userRole', headerName: '4', width: 80}
+    ];
+	
+	const rows =RAM.map(RAM);
+
```
Comment (dlt-v) [inline cdp1802app/src/RAMSYSTEM.js:46]: spaces

File: cdp1802app/src/RAMSYSTEM.js [hunk 11]
```diff
@@ -24,36 +24,33 @@ import {useState} from 'react';
...
+    ];
+	
+	const rows =RAM.map(RAM);
+	
+
+
+	
+	return (
+	
+	  <p aaa />
```
Comment (dlt-v) [inline cdp1802app/src/RAMSYSTEM.js:52]: ?

**Review by dlt-v — COMMENTED** [review]
Check comments for changes

**Comment by dlt-v** [comment]
Code not main branch ready, imo. Inconsistent, mix of pre-ES6 and new ES6+ code in same files.
Console logs are not for main branch. Leave them in debug before merge.
Please consider using a formatter to format your white spaces.
It would be useful to write some comments about what the code does.
Cheers,
- Delta

**Comment by MsAlPatina** [comment]
> Code not main branch ready, imo. Inconsistent, mix of pre-ES6 and new ES6+ code in same files.
> Console logs are not for main branch. Leave them in debug before merge.
> Please consider using a formatter to format your white spaces.
> It would be useful to write some comments about what the code does.
> Cheers,
> - Delta

Alrighty, maybe can ya suggest an good formatter?

**Comment by dlt-v** [comment]
> 

I use Prettier

**Pull request closed without merging**